### DISCUSSION TITLE
wixlib in nupkg

### DISCUSSION THREAD

**Discussion by olejniczak** [wixtoolset/issues#7324] [Questions]
I have common repetitive code (fragments with file components) in several installer projects.
I want to move them to the wixlib library.
Question: if and how to use nuget to distribute the library?
dotnet pack does not work for wixproj - error MSB4057

**Comment by robmen** [comment]
Correct. The `Pack` target was not implemented in WiX v4 .wixproj. Maybe someone would be interested in implementing it in a future version. In WiX v4, you can create the .nupkg by writing a custom target in the .wixproj or many other means.
